packages/browser/src/login/oidc/refresh/TokenRefresher.spec.ts

```diff
@@ -231,14 +235,15 @@ describe("TokenRefresher", () => {
       const refresher = getTokenRefresher({
         storageUtility: mockedStorage,
       });
-      const callback = jest.fn();
+
       await refresher.refresh(
         "mySession",
-        "some refresh token",
+        "some old refresh token",
         await mockKeyPair(),
-        callback
+        mockEmitter
       );
-      expect(callback).toHaveBeenCalled();
+
+      expect(mockEmit).toHaveBeenCalled();
```

Comment:
Here, I would test that the callback has been called with the appropriate arguments while we are at it, to verify that the actual refresh token value has been passed (it's writing this test for the node version that I realised I wasn't actually passing the refresh token value to the callback, hurray for test-driven development ^^)